pkgs/applications/misc/tiramisu/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "tiramisu";
-  version = "2.0.20211107";
+  version = "unstable-2023-03-29";
 
   src = fetchFromGitHub {
     owner = "Sweets";
     repo = pname;
-    rev = version;
-    sha256 = "1n1x1ybbwbanibw7b90k7v4cadagl41li17hz2l8s2sapacvq3mw";
+    rev = "5dddd83abd695bfa15640047a97a08ff0a8d9f9b";
+    sha256 = "sha256-owYk/YFwJbqO6/dbGKPE8SnmmH4KvH+o6uWptqQtpfI=";
```

```suggestion
    hash = "sha256-owYk/YFwJbqO6/dbGKPE8SnmmH4KvH+o6uWptqQtpfI=";
```

https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub

> hash is currently preferred

---

Totally missed that, thanks for noticing!